### PR TITLE
monitor: mark the starting point for clusteroperators

### PR DESCRIPTION
**Pull request by deads2k**
We recently had a PR that made the operator perma-degraded and CI didn't show it.

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/1073/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws-serial/1384229570948894720

### CODE REVIEW

File: pkg/monitor/operator.go [hunk 1]
```diff
@@ -76,11 +37,10 @@ func startClusterOperatorMonitoring(ctx context.Context, m Recorder, client conf
...
 				c := &co.Status.Conditions[i]
-				previous := findOperatorStatusCondition(oldCO.Status.Conditions, c.Type)
-				if previous == nil {
-					continue
-				}
-				if c.Status != previous.Status {
+				previousCondition := findOperatorStatusCondition(oldCO.Status.Conditions, c.Type)
+				// If we don't have a previous state, then we should always mark the starting state with an event.
+				// We recently had a PR that caused the kube-apiserver operator be permanently degraded and it didn't show up.
+				if previousCondition == nil || c.Status != previousCondition.Status {
```
Comment (tkashem) [inline pkg/monitor/operator.go:43]: do we want to log here that previous condition was missing so when we go through the log we have more context?

Comment (tkashem) [reply pkg/monitor/operator.go:43]: actually, we only log what we changed to, we don't log what we changed from. so this is in keeping with the current behavior.

Comment (deads2k) [reply pkg/monitor/operator.go:43]: > actually, we only log what we changed to, we don't log what we changed from. so this is in keeping with the current behavior.

correct

File: pkg/monitor/operator.go [hunk 2]
```diff
@@ -17,45 +17,6 @@ import (
 	configclientset "github.com/openshift/client-go/config/clientset/versioned"
 )
 
-// condition/Degraded status/True reason/DNSDegraded changed: DNS default is degraded
-func GetOperatorConditionStatus(message string) *configv1.ClusterOperatorStatusCondition {
```
Comment (wking) [inline pkg/monitor/operator.go:21]: bbd9949c834a14 (#26034) moved this to `pkg/monitor/monitorapi`.  :+1: to dropping this old, unused copy.

**Comment by deads2k** [comment]
only affects the monitor, bypassing bug check

**Pull request merged**